src/common/services/api/designations.service.spec.js

```diff
@@ -0,0 +1,33 @@
+import angular from 'angular';
+import 'angular-mocks';
+import module from './designations.service';
+
+import searchResponse from 'common/services/api/fixtures/product-search.fixture';
+
+describe('designation service', () => {
+  beforeEach(angular.mock.module(module.name));
+  var self = {};
+
+  beforeEach(inject((designationsService, $httpBackend) => {
+    self.designationsService = designationsService;
+    self.$httpBackend = $httpBackend;
+  }));
+
+  afterEach(() => {
+    self.$httpBackend.verifyNoOutstandingExpectation();
+    self.$httpBackend.verifyNoOutstandingRequest();
+  });
+
+  describe('productSearch', () => {
+    it('should send a request to API and get results', () => {
+      self.$httpBackend.expectGET('https://cortex-gateway-stage.cru.org/search?keyword=steve').respond(200, searchResponse);
+      self.designationsService.productSearch({
+        keyword: 'steve'
+      })
+        .subscribe((data) => {
+          expect(data.length).toEqual(searchResponse.response.docs.length);
```

Comment:
I think you should check to make sure at least the first one is mapped correctly and has the keys you expect.